components/immatriculations/index.tsx

```diff
@@ -51,6 +52,9 @@ const Immatriculations: React.FC<IProps> = ({
         immatriculationRNE={immatriculationRNE}
         uniteLegale={uniteLegale}
       />
+      <AvisSituationSection uniteLegale={uniteLegale} session={session} />
+      <HorizontalSeparator />
+
```

Comment:
Why move this up ? 

On a business point of view it does not really make sense as the association immatriculation is more equivalent to `immatriculationRNE` and therefore is expected to come first

Plus I expect most people would open thise page for the `immatriculation` rather than the `AvisSituation`

Reply:
I explained it in the commit : 
```
In `Dirigeants` section, we display the `Avis de situation` section
before the `RNE` section. The reason is that the inpi often takes very
long to load. In order to improve UX, we display the `Avis de situation`
which is already loaded. This means that the UI changes happens outside
of the viewport.
```

But I understand the point about people wanting to see the immatriculation RNE first. Besides, it changes the way things are presented which can be annoying for people that are used to it. I let you decide if you think it's better to skip this commit I'm totally OK.